src/lib/modules/ESSelector/HitsSearch.js

```diff
@@ -162,6 +158,13 @@ export class HitsSearch extends Component {
     );
   };
 
+  handleKeyPress = async (event) => {
+    const { onResults } = this.props;
+    if (event.key === 'Enter' && onResults !== undefined) {
+      this.setState({ execute: true });
```

Comment:
Was this solution of adding a new `state` discussed? Can we chat about this IRL?

Reply:
As discussed IRL, explore the option of bubbling up the `onKeyPress` event, and implement the login of checking-out when clicking `Enter` in the `ItemActionMenu`: this would move the extra new state up, closer to the feature where it is needed.